cli/tests/integrations/test_dcos.py

```diff
@@ -27,50 +27,9 @@ def test_default():
 
 
 def test_help():
-    stdout = b"""\
-Command line utility for the Mesosphere Datacenter Operating
-System (DCOS)
-
-'dcos help' lists all available subcommands. See 'dcos <command> --help'
-to read about a specific subcommand.
-
-Usage:
-    dcos [options] [<command>] [<args>...]
-
-Options:
-    --help                      Show this screen
-    --version                   Show version
-    --log-level=<log-level>     If set then print supplementary messages to
-                                stderr at or above this level. The severity
-                                levels in the order of severity are: debug,
-                                info, warning, error, and critical. E.g.
-                                Setting the option to warning will print
-                                warning, error and critical messages to stderr.
-                                Note: that this does not affect the output sent
-                                to stdout by the command.
-    --debug                     If set then enable further debug messages which
-                                are sent to stdout.
-
-Environment Variables:
-    DCOS_LOG_LEVEL              If set then it specifies that message should be
-                                printed to stderr at or above this level. See
-                                the --log-level option for details.
-
-    DCOS_CONFIG                 This environment variable points to the
-                                location of the DCOS configuration file.
-                                [default: ~/.dcos/dcos.toml]
-
-    DCOS_DEBUG                  If set then enable further debug messages which
-                                are sent to stdout.
-
-    DCOS_SSL_VERIFY             If set, specifies whether to verify SSL certs
-                                for HTTPS, or the path to the certificate(s).
-                                Can also be configured by setting
-                                `core.ssl_config` in the config.
-"""
-
-    assert_command(['dcos', '--help'],
-                   stdout=stdout)
+    with open('tests/data/help/main.txt') as content:
```

Comment:
Let's change this to `.../dcos.txt` if we rename the help file for the root dcos command.